docker/Dockerfile

```diff
@@ -154,6 +154,13 @@ RUN python install.py -j ${make_cores} --build-type=Release --core-version 99999
 ENV PATH /root/.local/bin:$PATH
 ENV LD_LIBRARY_PATH /root/.local/lib:/root/.local/lib/cyclus
 
+FROM cyclus as deb-generation
+WORKDIR /cyclus/build
+RUN make package
+
+FROM scratch as deb-package
+COPY --from=deb-generation /cyclus/build/cyclus*.deb /
```

Comment:
What does this target accomplish?

Reply:
Don't we want to save this as an artifact somewhere?

Reply:
This is how we can pass the .deb file back out to the GitHub runner.  Otherwise it only exists within the docker image and we cannot access it on the runner.  The code coverage workflow follows a similar pattern to pass coverage reports back out to the client.  The files contained in this `scratch` layer are outputted to a destination that we specify when we build the target - see line 76 of `publish_release.yml`